lexicons/app/bsky/feed/viewItem.json

```diff
@@ -0,0 +1,48 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.postInFeed",
```

Comment:
Bikeshed! I feel like `feedViewPost` is a little less clunky?

Reply:
Likewise: `threadViewPost`

Reply:
yup i like it 👍